test.js

```diff
@@ -533,18 +533,18 @@ test('alias - require previousId and userId', (t) => {
 test('isErrorRetryable', (t) => {
   const client = createClient();
 
-  t.false(client.isErrorRetryable({}));
-
-  // ETIMEDOUT is retryable as per `is-retry-allowed` (used by axios-retry in `isNetworkError`).
-  t.true(client.isErrorRetryable({ code: 'ETIMEDOUT' }));
-
-  // ECONNABORTED is not retryable as per `is-retry-allowed` (used by axios-retry in `isNetworkError`).
-  t.false(client.isErrorRetryable({ code: 'ECONNABORTED' }));
-
-  t.true(client.isErrorRetryable({ response: { status: 500 } }));
-  t.true(client.isErrorRetryable({ response: { status: 429 } }));
-
-  t.false(client.isErrorRetryable({ response: { status: 200 } }));
+  // test error cases
+  t.true(client.isErrorRetryable(0, {}, {}));
+  t.true(client.isErrorRetryable(0, { code: 'ETIMEDOUT' }, {}));
+  t.true(client.isErrorRetryable(0, { code: 'ECONNABORTED' }, {}));
```

Comment:
Currently, we are retrying on all network related errors, which seems fine to me. If we do want to replicate the rejection on `ECONNABORTED` I'm happy to add that in as well.